File: Python/Ex.9.py
matriz = [[None, None, None], [None, None, None]]

for l in range(0,2):
  for c in range(0,3):
    if l == 0 and c == 0:
      matriz[l][c] = int(input("Insira um número: "))
    else:
      if c == 0:
        matriz[l][c] = matriz[l - 1][2] * 7
      else:
        matriz[l][c] = matriz[l][c - 1] * 7

print(matriz)

File: JavaScript/Ex.9.js
let matriz = [[0,0,0], [0,0,0]]

for(let l = 0; l < 2; l++){
  for(let c = 0; c < 3; c++){
    if(l == 0 && c == 0){
      matriz[l][c] = Number(prompt("Insira um número: "))
    } else {
      if(c == 0){
        matriz[l][c] = matriz[l - 1][2] * 7
      } else {
        matriz[l][c] = matriz[l][c - 1] * 7
      }
    }
  }
}
console.log(matriz)